### ISSUE TITLE
[Bug]: exposes.composite don't show values in frontend

### ISSUE THREAD

**Issue by Burki24** [nurikk/z2m-frontend#1535]
### What happened?

Zigbee2mqtt v. 1.27.0-dev commit: [fea5d44a]
z2m-Frontend v. 0.6.107
RPI4 4GB
Symcon Home automation system (www.symcon.de)
Sonoff Zigbee-Stick 3.0+ last dev Stack

When I expand the exposes from this device: https://www.zigbee2mqtt.io/devices/511.344.html getting the following problem: The payload shows details for the action expose:
```json
{
"action":"color_move_ep2",
"action_color":{
    "x":0.36,
    "y":0.632},
"action_group":61954,
"action_transition_time":0,
"battery":5,
"linkquality":36}
```
I want to add these to the converter (action_group, action_transition_time, action_color)

But "action_color" is a composite payload. So i did following additions to the converter:
```txt
e.action_group(),
            exposes.numeric('action_transition_time', ea.STATE),
            exposes.numeric('action_color_temperature', ea.STATE).withUnit('mired'),
            exposes.composite('action_color', 'action_color')
                .withFeature(exposes.numeric('x', ea.STATE))
                .withFeature(exposes.numeric('y', ea.STATE)),  
```
All exposes are in the frontend. But action_color is not  filled with values:

![image](https://user-images.githubusercontent.com/20765982/184553855-c1d328ef-5726-4eca-b8c7-f8fcd35f79bc.png)

I asked Koenk (https://github.com/Koenkk/zigbee2mqtt/discussions/13544) he said, I have to ask here for a solution.
In state.json it shows:

```json
"0x94deb8fffe796989": {
            "date_code": "NULL",
            "definition": {
                "description": "Zigbee handheld remote RGBW 4 channels",
                "exposes": [
                    {
                        "access": 1,
                        "description": "Remaining battery in %",
                        "name": "battery",
                        "property": "battery",
                        "type": "numeric",
                        "unit": "%",
                        "value_max": 100,
                        "value_min": 0
                    },
                    {
                        "access": 1,
                        "description": "Triggered action (e.g. a button click)",
                        "name": "action",
                        "property": "action",
                        "type": "enum",
                        "values": [
                            "color_move",
                            "color_temperature_move",
                            "hue_move",
                            "hue_stop",
                            "brightness_step_up",
                            "brightness_step_down",
                            "recall_*",
                            "on",
                            "off"
                        ]
                    },
                    {
                        "features": [
                            {
                                "access": 1,
                                "name": "x",
                                "property": "x",
                                "type": "numeric"
                            },
                            {
                                "access": 1,
                                "name": "y",
                                "property": "y",
                                "type": "numeric"
                            }
                        ],
                        "name": "action_color",
                        "property": "action_color",
                        "type": "composite"
                    },
                    {
                        "access": 1,
                        "name": "action_group",
                        "property": "action_group",
                        "type": "numeric"
                    },
                    {
                        "access": 1,
                        "name": "action_transition_time",
                        "property": "action_transition_time",
                        "type": "numeric"
                    },
                    {
                        "access": 1,
                        "description": "Link quality (signal strength)",
                        "name": "linkquality",
                        "property": "linkquality",
                        "type": "numeric",
                        "unit": "lqi",
                        "value_max": 255,
                        "value_min": 0
                    }
                ],
                "model": "511.344",
                "options": [
                    {
                        "description": "Simulate a brightness value. If this device provides a brightness_move_up or brightness_move_down action it is possible to specify the update interval and delta.",
                        "features": [
                            {
                                "access": 2,
                                "description": "Delta per interval, 20 by default",
                                "name": "delta",
                                "property": "delta",
                                "type": "numeric",
                                "value_min": 0
                            },
                            {
                                "access": 2,
                                "description": "Interval duration",
                                "name": "interval",
                                "property": "interval",
                                "type": "numeric",
                                "unit": "ms",
                                "value_min": 0
                            }
                        ],
                        "name": "simulated_brightness",
                        "property": "simulated_brightness",
                        "type": "composite"
                    }
                ],
                "supports_ota": false,
                "vendor": "Iluminize"
            },
            "endpoints": {
                "1": {
                    "bindings": [],
                    "clusters": {
                        "input": [
                            "genBasic",
                            "genPowerCfg",
                            "genIdentify",
                            "haDiagnostic",
                            "touchlink"
                        ],
                        "output": [
                            "genIdentify",
                            "genGroups",
                            "genScenes",
                            "genOnOff",
                            "genLevelCtrl",
                            "genOta",
                            "lightingColorCtrl",
                            "touchlink"
                        ]
                    },
                    "configured_reportings": [],
                    "scenes": []
                },
                "2": {
                    "bindings": [],
                    "clusters": {
                        "input": [
                            "genBasic",
                            "genPowerCfg",
                            "genIdentify",
                            "haDiagnostic",
                            "touchlink"
                        ],
                        "output": [
                            "genIdentify",
                            "genGroups",
                            "genScenes",
                            "genOnOff",
                            "genLevelCtrl",
                            "genOta",
                            "lightingColorCtrl",
                            "touchlink"
                        ]
                    },
                    "configured_reportings": [],
                    "scenes": []
                },
                "3": {
                    "bindings": [],
                    "clusters": {
                        "input": [
                            "genBasic",
                            "genPowerCfg",
                            "genIdentify",
                            "haDiagnostic",
                            "touchlink"
                        ],
                        "output": [
                            "genIdentify",
                            "genGroups",
                            "genScenes",
                            "genOnOff",
                            "genLevelCtrl",
                            "genOta",
                            "lightingColorCtrl",
                            "touchlink"
                        ]
                    },
                    "configured_reportings": [],
                    "scenes": []
                },
                "4": {
                    "bindings": [],
                    "clusters": {
                        "input": [
                            "genBasic",
                            "genPowerCfg",
                            "genIdentify",
                            "haDiagnostic",
                            "touchlink"
                        ],
                        "output": [
                            "genIdentify",
                            "genGroups",
                            "genScenes",
                            "genOnOff",
                            "genLevelCtrl",
                            "genOta",
                            "lightingColorCtrl",
                            "touchlink"
                        ]
                    },
                    "configured_reportings": [],
                    "scenes": []
                }
            },
            "friendly_name": "0x94deb8fffe796989",
            "ieee_address": "0x94deb8fffe796989",
            "interview_completed": true,
            "interviewing": false,
            "manufacturer": "iluminize",
            "model_id": "ZG2819S-RGBW",
            "network_address": 3826,
            "power_source": "Battery",
            "software_build_id": "2.5.3_r21",
            "supported": true,
            "type": "EndDevice"
        },
```

### What browsers are you seeing the problem on?

Firefox, Chrome, Microsoft Edge

### Did you download state.json.zip?

I will attach state.json.zip

**Comment by Burki24** [comment]
Sorry for delay. As Attachement, you'll find state.json.zip
[state.json.zip](https://github.com/nurikk/zigbee2mqtt-frontend/files/9347952/state.json.zip)

**Comment by Burki24** [comment]
Fixed in Frontend 0.6.108, Thanks @nurikk
